packages/scripts/config/webpack.config.js

```diff
@@ -154,6 +154,18 @@ module.exports = (webpackEnv) => {
             ),
           },
         },
+        {
+          test: /web\/src\/components\/.+Cell.js$/,
```

Comment:
This feels a bit hacky, but gets the job done. Probably we could tighten this up a bit more.